### PR TITLE
Updating UK (19th Sept 2024 ClearBank Changes)

### PR DESCRIPTION
**Pull request by Orkan-M**
Updates UK wording.

### CODE REVIEW

File: en-gb/docs/API Overview/pis.md [hunk 1]
```diff
@@ -20,10 +20,12 @@ ClearBank's Open Banking API powered by Ozone **does not** support:
 The following fields are mandatory for all domestic payment consents:
 
 ### `InstructedAmount/Amount`
-The ClearBank Open Banking API mandates no maximum amount in `InstructedAmount/Amount`. It is possible that a `domestic-payment-consents` is authorised, but the payment initiation fails due to account limits.
+The ClearBank Open Banking API mandates a maximum amount in `InstructedAmount/Amount`. As CleearBank will be processing your payments through Faster Payments (FPS), ClearBank mandates a maximum amount per payment of £1,000,000. ClearBank recommends that PISP notify the PSU of the limits applied on payments.
```
Comment (mattregis) [inline en-gb/docs/API Overview/pis.md:23]: ClearBank

**Review by mattregis — COMMENTED** [review]
ready to approve, one minor typo to update

File: en-gb/docs/10-index.md [hunk 2]
```diff
@@ -1,39 +1,23 @@
...
-
-```
-
 ## ClearBank API Documentation
 
-ClearBank’s Open Banking services powered by Ozone follows the Open Banking specifications found here:
+ClearBank’s Open Banking services powered by Ozone follow the Open Banking specifications:
 
-- [Dynamic Client Registration (DCR) Specification](https://openbankinguk.github.io/dcr-docs-pub/v3.2/dynamic-client-registration.html): Use this specification to register your TPP client to use our APIs
+- [Dynamic Client Registration (DCR) Specification](https://openbankinguk.github.io/dcr-docs-pub/v3.2/dynamic-client-registration.html): This specification allows to register your TPP client to use our APIs.
```
Comment (lscancb) [inline en-gb/docs/10-index.md:7]:
```suggestion
- [Dynamic Client Registration (DCR) Specification](https://openbankinguk.github.io/dcr-docs-pub/v3.2/dynamic-client-registration.html): This specification allows you to register your TPP client to use our APIs.
```

File: en-gb/docs/API Overview/pis.md (hunk 1 above)
Comment (Orkan-M) [reply en-gb/docs/API Overview/pis.md]: Updated now

**Review by lscancb — APPROVED** [review]

**Review by mattregis — APPROVED** [review]

**Pull request merged**